sql-cli/sql_cli/__main__.py

```diff
@@ -85,8 +85,18 @@ def validate(
 ) -> None:
     project_dir_absolute = project_dir.resolve() if project_dir else Path.cwd()
     project = Project(project_dir_absolute)
+    project.load_config(environment=env)
 
-    validate_connections(project=project, environment=env, connection_id=connection)
+    connections = [convert_to_connection(c) for c in project.connections]
```

Comment:
At this point should we just convert to connections at load from Yaml time? Do we have use for the connection dictionaries anymore?